storage/storage_discoverHint.go

```diff
@@ -0,0 +1,72 @@
+package storage
+
+import (
+	"github.com/bmc-toolbox/dora/filter"
+	"github.com/bmc-toolbox/dora/model"
+	"github.com/jinzhu/gorm"
+)
+
+// NewDiscoverHintStorage initializes the storage
+func NewDiscoverHintStorage(db *gorm.DB) *DiscoverHintStorage {
+	return &DiscoverHintStorage{db}
+}
+
+// DiscoverHintStorage stores all DiscoverHints
+type DiscoverHintStorage struct {
+	db *gorm.DB
+}
+
+// Count gets DiscoverHints count based on the filter
+func (s DiscoverHintStorage) Count(filters *filter.Filters) (count int, err error) {
+	q, err := filters.BuildQuery(model.DiscoverHint{}, s.db)
+	if err != nil {
+		return count, err
+	}
+
+	err = q.Model(&model.DiscoverHint{}).Count(&count).Error
+	return count, err
+}
+
+// GetAll of the DiscoverHints
+func (s DiscoverHintStorage) GetAll(offset string, limit string) (count int, ips []model.DiscoverHint, err error) {
+	if offset != "" && limit != "" {
```

Comment:
Please double-check the logic you want is `&&` and not `||`.
Currently will use the limit & offset only when both are non-"" and otherwise use neither.
Meaning `discoverHint.GetAll("","10")` would give you all hints - not limit to (first) 10 like you might expect.

Reply:
Fixed